### ISSUE TITLE
Library request: livelossplot

### ISSUE THREAD

**Issue by AlexCatarino** [QuantConnect/Lean#6685]
[livelossplot](https://p.migdal.pl/livelossplot/): A live training loss plot in [Jupyter Notebook](http://jupyter.org/) for [Keras](https://keras.io/), [PyTorch](http://pytorch.org/) and other frameworks.

#### Checklist
- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)
